scipy/stats/_binned_statistic.py

```diff
@@ -392,7 +392,8 @@ def binned_statistic_dd(sample, values, statistic='mean',
           * 'sum' : compute the sum of values for points within each bin.
             This is identical to a weighted histogram.
           * 'std' : compute the standard deviation within each bin. This
-            is implicitly calculated with ddof=0.
+            is implicitly calculated with ddof=0. If the bin number is 0 or 1,
```

Comment:
```suggestion
            is implicitly calculated with ddof=0. If the number of values within a given bin is 0 or 1, 
the computed standard deviation value will be 0 for the bin.
```

if you agree the layout of the suggestion across the lines might need editing to keep within PEP8.